### PR TITLE
[Mediapipe] Sync to release 0.8.10.2 (Windows)

### PR DESCRIPTION
**Pull request by christoph-nimagna**
- merged/synced with [official Mediapipe 0.8.10.2 release](https://github.com/google/mediapipe/releases/tag/v0.8.10.2)
- Added batch file simplifying building the DLL
- Removed unneeded dependencies and includes
- added `/FORCE:MULTIPLE` linker option to prevent from linker errors due to duplicate symbols
- Adapted README

### CODE REVIEW

File: mediapipe/pose_tracking_dll/pose_tracking.cpp [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
+//#include "mediapipe/framework/formats/image_frame.h"
 #include "mediapipe/framework/formats/image_frame_opencv.h"
 #include "mediapipe/framework/formats/landmark.pb.h"
 #include "mediapipe/framework/port/file_helpers.h"
-#include "mediapipe/framework/port/opencv_highgui_inc.h"
-#include "mediapipe/framework/port/opencv_imgproc_inc.h"
-#include "mediapipe/framework/port/opencv_video_inc.h"
+//#include "mediapipe/framework/port/opencv_highgui_inc.h"
+//#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+//#include "mediapipe/framework/port/opencv_video_inc.h"
```
Comment (valix25) [inline mediapipe/pose_tracking_dll/pose_tracking.cpp:31]: The includes can be removed if not needed.

File: mediapipe/pose_tracking_dll/pose_tracking.cpp [hunk 2]
```diff
@@ -22,13 +22,13 @@
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
 #include "mediapipe/framework/calculator_framework.h"
-#include "mediapipe/framework/formats/image_frame.h"
+//#include "mediapipe/framework/formats/image_frame.h"
```
Comment (valix25) [inline mediapipe/pose_tracking_dll/pose_tracking.cpp:25]: Can be removed if not needed.

File: mediapipe/pose_tracking_dll/BUILD [hunk 3]
```diff
@@ -31,26 +31,19 @@ windows_dll_library(
...
-        "//mediapipe/calculators/tflite:tflite_model_calculator",
-        "//mediapipe/calculators/util:local_file_contents_calculator",
         "//mediapipe/graphs/pose_tracking:pose_tracking_cpu_deps",
     ]
 )
 
 # **Implicitly link to face_mesh_lib.dll**
 cc_binary(
     name = "pose_tracking_cpu",
+	linkopts = ["/FORCE:MULTIPLE"],
```
Comment (valix25) [inline mediapipe/pose_tracking_dll/BUILD:46]: I forgot what was the cause of having duplicate symbols.

**Review by valix25 — CHANGES_REQUESTED** [review]

File: mediapipe/pose_tracking_dll/pose_tracking.cpp (hunk 1 above)
Comment (christoph-nimagna) [reply mediapipe/pose_tracking_dll/pose_tracking.cpp:31]: 👍

File: mediapipe/pose_tracking_dll/BUILD (hunk 3 above)
Comment (christoph-nimagna) [reply mediapipe/pose_tracking_dll/BUILD:46]: I added a comment explaining the cause

**Review by valix25 — APPROVED** [review]
Can be merged!

**Pull request closed without merging**